setup.py

```diff
@@ -1,7 +1,6 @@
 import sys, glob, os, subprocess
 
-major, minor1, minor2, s, tmp = sys.version_info
-if major==2 and minor1<4 or major<2:
+if sys.version_info < (2, 4):
```

Comment:
I was just trying to make the version requirements clearer without having all of those extra variables.  A minimum version of 2.6 is reasonable  Its last security fix, 2.6.9, was on October 29, 2013.

I'll modify that line to 2.6.